builds/fabric8-che/assembly/assembly-wsagent-war/pom.xml

```diff
@@ -145,71 +55,4 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>prepare-patch-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                    <execution>
-                        <id>copy-patched-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>build-helper-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>add-source-for-patched-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                    <execution>
-                        <id>add-resource-for-patched-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <artifactId>maven-resources-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>copy-web-resources</id>
-                        <phase>process-sources</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-patch-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>patch-sources</id>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-war-plugin</artifactId>
```

Comment:
What about the question about Keycloak ? The inherited war configuration changes the `warSourceDirectory` to `${project.build.directory}/patched/src/main/webapp`.

That's why the `copy-web-resources` execution of the `maven-resources-plugin` plugin was useful, for example to include `WEB-INF/classes/codenvy/rh-che-machine-configuration.properties` or `WEB-INF/keycloak.json`. As far as I can see in the build result they are not in the generated war.